ee/desktop/user/universallink/handler.m

```diff
@@ -0,0 +1,19 @@
+//go:build darwin
+// +build darwin
+
+#import <Foundation/Foundation.h>
+
+BOOL registerAppBundle(char *cAppBundlePath) {
```

Comment:
Do we need to free/release anything in the C side?

Reply:
The string is freed on 106 in handler_darwin.go -- is there anything else I'm missing?

Reply:
I don't know enough to know if we need to free `appBundlePath` or `url` or `status`

Reply:
Ah, no, I don't believe we need to! We don't have ownership of those objects. [This](https://stackoverflow.com/a/8702692) has a clear overview.

Reply:
excellent. Thank you!